src/lpurge.c

```diff
@@ -610,6 +610,57 @@ purge(const char *path, time_t thresh, struct elist_struct *elist,
         return inode_count;
 }
 
+#ifdef HAVE_LINUX_LUSTRE_LUSTRE_USER_H
+
+/* Lustre >= 2.12 */
+static int
+llapi_stat_mds(int fd, char *fname, struct stat *sb)
+{
+        const size_t bufsize = MAX(MAXPATHLEN,
+                                   sizeof(struct lov_user_mds_data) +
+                                   sizeof(struct lov_user_ost_data_v1) * 2000);
```

Comment:
Nitpick: Should this be "lov_user_ost_data" to be consistent with dropping the version from "lov_user_mds_data"?

Reply:
Yes, that makes sense.  Fixed.